### PR TITLE
Double click to open referrers

### PR DESCRIPTION
**Pull request by etbyrd**
Fix for https://github.com/drewnoakes/string-theory/issues/10

### CODE REVIEW

File: StringTheory/UI/StringList.xaml [hunk 1]
```diff
@@ -86,6 +86,12 @@
                 <DataGridTextColumn Width="Auto" ElementStyle="{StaticResource StatisticStyle}" Header="LOH%"   Binding="{Binding LohPercent,  Converter={x:Static local:PercentageConverter.Default}}" local:DataGridExtensions.SortDesc="True" />
             </DataGrid.Columns>
 
+            <DataGrid.Resources>
+                <Style TargetType="DataGridRow">
+                    <EventSetter Event="MouseDoubleClick" Handler="DataGridRow_DoubleClick"/>
```
Comment (drewnoakes) [inline StringTheory/UI/StringList.xaml]: Adding event handlers (`DataGridRow_DoubleClick` here) in WPF is generally frowned upon. When the UI wants to invoke code in the model, binding to an `ICommand` is usually preferrable.

We already have a command for showing referrers, and it's bound to the context menu item. You can reuse it here, and add an entry to `Grid.InputBindings` on line 75. Something like:

```xml
<MouseBinding Gesture="LeftDoubleClick" Command="{Binding ShowReferrersCommand}" />
```

With that, we shouldn't have to add any code behind, which is a benefit of commands.

**Comment by drewnoakes** [comment]
Closed/reopened in the hope of triggering a CI build. Didn't seem to work.

**Comment by bgrainger** [comment]
/AzurePipelines run

https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops#trigger-builds-using-github-pull-request-comments

**Comment by drewnoakes** [comment]
Addressed in 68fde9b2c6bd21dc49c25ebfbf2a1a5fb1b3db6d.

**Pull request closed without merging**